python/lsst/analysis/tools/tasks/sourceObjectTableAnalysis.py

```diff
@@ -85,6 +223,11 @@ class SourceObjectTableAnalysisConfig(
         dtype=str,
         default="r_dec",
     )
+    epoch_column = pexConfig.Field(
+        doc="Name of column in refCat to use for epoch.",
+        dtype=str,
+        default="r_epoch",
```

Comment:
This will then always use the `r`-band epoch? That doesn't seem correct.

Reply:
It's in the SRD.

Reply:
Hmmm, that's worrying to me, since the epochs won't be the same between bands. Unless we plan to shift everything to a single epoch and store all of our values relative to that? I might be confused about how this will be used.

Reply:
They are different between bands, which is why the `objectEpochTableTask` goes through the healsparse maps for each band, and then I'm grabbing the r-band epoch here, because I am going to compare the source positions to the r-band object positions.